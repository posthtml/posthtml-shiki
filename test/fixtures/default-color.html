<shiki lang="js" theme-light="github-light" theme-dark="github-dark">
  import { codeToHtml } from 'shiki'
</shiki>
