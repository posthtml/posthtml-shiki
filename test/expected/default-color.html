<pre class="shiki shiki-themes github-dark github-light" style="background-color:#24292e;--shiki-light-bg:#fff;color:#e1e4e8;--shiki-light:#24292e" tabindex="0"><code><span class="line"></span>
<span class="line"><span style="color:#F97583;--shiki-light:#D73A49">  import</span><span style="color:#E1E4E8;--shiki-light:#24292E"> { codeToHtml } </span><span style="color:#F97583;--shiki-light:#D73A49">from</span><span style="color:#9ECBFF;--shiki-light:#032F62"> 'shiki'</span></span>
<span class="line"></span></code></pre>
